tests/image-size.bs

```diff
@@ -0,0 +1,35 @@
+<pre class=metadata>
```

Comment:
Add a `001` to the end of the filename, to match the pattern elsewhere, and make it easy to add more if needed later.

Reply:
Also, I think I'd prefer the test and support files grouped into a folder of their own, like the `adjacent-boilerplate` test currently does. (I know I'm not consistent on this matter currently.)

Reply:
done